examples/verif-manual/vm-013.py

```diff
@@ -0,0 +1,152 @@
+"""
+VM13 Cylindrical Shell Under Pressure
+=========================================
+
+Description:
+A long cylindrical pressure vessel of mean diameter d and wall thickness t has closed
+ends and is subjected to an internal pressure P. Determine the axial stress
+:math:`/sigma _y` and the hoop stress :math:`/sigma _z` in the vessel at the
+midthickness of the wall.
```

Comment:
To follow the same format as for the other exposed VM files, the description present in the MAPDL documentation needs to be added to each example.